inc/ThirdParty/Plugins/PageBuilder/Elementor.php

```diff
@@ -389,14 +389,8 @@ public function clear_action() {
 			return;
 		}
 
-		$url = wp_get_referer();
-
-		if ( 0 !== strpos( $url, 'http' ) ) {
-			$parse_url = get_rocket_parse_url( untrailingslashit( home_url() ) );
-			$url       = $parse_url['scheme'] . '://' . $parse_url['host'] . $url;
-		}
-
-		$this->used_css->clear_url_usedcss( $url );
+		$this->used_css->delete_all_used_css();
+		$this->used_css->delete_used_css_rows();
```

Comment:
```suggestion
		$this->used_css->delete_used_css_rows();
```
`delete_used_css_rows()` already calls `delete_all_used_css()` inside it